mule-shaders-module/pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.mule.extensions</groupId>
+        <artifactId>mule-core-modules-parent</artifactId>
+        <version>1.1.9</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.mule.modules</groupId>
+    <artifactId>mule-shaders-module</artifactId>
+    <version>1.0.0-rc1</version>
+    <packaging>jar</packaging>
+    <name>Shader Utilities Module</name>
```

Comment:
It is technically versioned independently of the JSON Module, since we're using `1.0.0-rc1`, but the version is more of a placeholder than anything else, since the artifact is not published anywhere and its just installed during the build process of the JSON Module as needed.

The reason why we even needed a separate module for custom shading has to do with classpath access limitations on some Maven plugins like the shade plugin, where custom shading or transformer classes can't be loaded by the plugin and have to be loaded from a module instead.

Do you think it's acceptible to structure the code this way, or do you think what we need to manage this artifact separately and have a separate release for it before JSON Module?